js/preload/preloadMain.js

```diff
@@ -416,6 +421,23 @@ function createAPI() {
         }
     });
 
+    /**
+     * Initializes snack bar
+     */
+    local.ipcRenderer.on('init-snack-bar', () => {
```

Comment:
why necessary to have init-snack-bar?  why not just lazy create???  is this some perf issue?